lib/download.ex

```diff
@@ -42,11 +43,18 @@ defmodule Download do
         do: { :ok, path }
   end
 
-  defp get_default_download_path(file_name) do
-    System.cwd() <> "/" <> file_name
+  defp get_default_dir() do
```

Comment:
This stuff is specific for your project, not for everyone. IE, `wget`, `rsync`, `curl` doesn't work as you specified